client/client.go

```diff
@@ -579,6 +799,48 @@ func (c *Client) Call(ctx context.Context, procedure string, options wamp.Dict,
 
 	switch msg := msg.(type) {
 	case *wamp.Result:
+
+		// Check and handle Payload PassThru Mode
+		// @see https://wamp-proto.org/wamp_latest_ietf.html#name-payload-passthru-mode
+		if pptScheme, _ := msg.Details[wamp.OptPPTScheme].(string); pptScheme != "" {
+			// Let's check: was ppt feature announced by dealer?
+			if !c.sess.HasFeature(wamp.RoleDealer, wamp.FeaturePayloadPassthruMode) {
+				// It's protocol violation, so we need to abort connection
+				abortMsg := wamp.Abort{
+					Reason: wamp.ErrProtocolViolation,
+					Details: wamp.Dict{
+						wamp.OptError:   ErrPPTNotSupportedByRouter.Error(),
+						wamp.OptMessage: ErrPPTNotSupportedByPeer.Error(),
+					},
+				}
+				c.sess.Peer.Send(&abortMsg)
+				c.sess.Peer.Close()
+				return nil, ErrPPTNotSupportedByRouter
```

Comment:
this error is indistinguishable from https://github.com/KSDaemon/nexus/pull/1/files#diff-bd3a55a72186f59e2e63efb4951573b2f9e4a7cc98086e922b0859f8ccc1dd09R756
maybe it should carry more context:
```suggestion
				return nil, fmt.Errorf("cannot process unexpected passthrough result: %v", ErrPPTNotSupportedByRouter)
```
this could use `%w`, but I'm not sure about the Go version nexus is targeting.